src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -169,6 +219,11 @@ public void WriteCritical(string message)
             }
         }
 
+        /// <summary>
+        /// Writes a critical event log without generating an event.
```

Comment:
should be same "summary" as a above. Only difference is 'formatting parameters' are passed.